alphadia/data/bruker.py

```diff
@@ -586,6 +586,16 @@ def assemble_push(
             dtype=np.float32,
         )
 
+        # intensities below HIGH_EPSILON will be set to zero
```

Comment:
there's a similar code in `AlphaRawJIT` .. maybe introduce two little helpers methods
`lower_zero_cutoff` and `zero_safe_division` ?

note there's also an alternative idiom https://stackoverflow.com/a/68118106

Reply:
In every other part of the code I would agree. But this is quite performance critical, so I want to have an explicit compile time constant.

Reply:
I doubt that importing this as a method would give any performance overhead here :-)